Jenkinsfile

```diff
@@ -55,6 +55,25 @@ pipeline {
                 }
             }
         }
+        stage('Docker iteration') {
+            when{
+                branch pattern: 'iteration\\d+', comparator: 'REGEXP'
```

Comment:
No "anyOf" needed?

Reply:
I don't think so, since it's only a single branch statement